config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-periodics-main.yaml

```diff
@@ -187,9 +187,9 @@ periodics:
     preset-azure-cred-only: "true"
     preset-azure-anonymous-pull: "true"
   extra_refs:
-    - org: kubernetes-sigs
+    - org: jackfrancis
```

Comment:
we should be running the same tests as presubmit jobs, is there any reason we can't use CI running on a PR to investigate failures? are there flakes observed in periodic jobs that can't be reproduced in presubmit?